<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin</class>
 <widget class="QMainWindow" name="MainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWin</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>467</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="m_pActionOpenFile"/>
    <addaction name="m_pActionSaveFile"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>Device</string>
    </property>
    <addaction name="m_pActionController"/>
    <addaction name="m_pActionUniverse"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDevice"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_pActionOpenFile"/>
   <addaction name="m_pActionSaveFile"/>
   <addaction name="separator"/>
   <addaction name="m_pActionController"/>
   <addaction name="m_pActionUniverse"/>
  </widget>
  <action name="m_pActionUniverse">
   <property name="icon">
    <iconset resource="../Resource/qLights.qrc">
     <normaloff>:/qLights/universe.png</normaloff>:/qLights/universe.png</iconset>
   </property>
   <property name="text">
    <string>Universe</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="m_pActionOpenFile">
   <property name="icon">
    <iconset resource="../Resource/qLights.qrc">
     <normaloff>:/Std_32x32/file_open.png</normaloff>:/Std_32x32/file_open.png</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="m_pActionSaveFile">
   <property name="icon">
    <iconset resource="../Resource/qLights.qrc">
     <normaloff>:/Std_32x32/file_save.png</normaloff>:/Std_32x32/file_save.png</iconset>
   </property>
   <property name="text">
    <string>Save File...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="m_pActionController">
   <property name="icon">
    <iconset resource="../Resource/qLights.qrc">
     <normaloff>:/qLights/controller.png</normaloff>:/qLights/controller.png</iconset>
   </property>
   <property name="text">
    <string>Controller</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Resource/qLights.qrc"/>
 </resources>
 <connections/>
</ui>
